windows/win_chocolatey.py

```diff
@@ -117,4 +118,9 @@
   win_chocolatey:
     name: ARRv3
     source: webpi
+
+  #Install git using an alternate source:
+  win_chocolatey:
```

Comment:
Ah I got you. Which is the next version? How can I tell when it will be released?